examples/gnrc_lorawan/Makefile

```diff
@@ -13,7 +13,9 @@ USEMODULE += netdev_default
 USEMODULE += auto_init_gnrc_netif
 
 # Add support for GNRC LoRaWAN
-USEMODULE += gnrc_lorawan
+#USEMODULE += gnrc_lorawan
+# Add support for GNRC LoRaWAN 1.1
+USEMODULE += gnrc_lorawan_1_1
```

Comment:
Are `gnrc_lorawan` and `gnrc_lorawan_1_1` mutually exclusive?

Reply:
No, `gnrc_lorawan_1_1` depends on `gnrc_lorawan`